.storybook/main.js

```diff
@@ -7,4 +12,17 @@ module.exports = {
     "@snek-at/storybook-addon-chakra-ui",
     "storybook-dark-mode",
   ],
+  webpackFinal: (config) => {
+    return merge(config, {
+      resolve: {
+        fallback: {
+          crypto: require.resolve('crypto-browserify'),
+          http: require.resolve('stream-http'),
+          https: require.resolve('https-browserify'),
+          stream: require.resolve('stream-browserify'),
+          zlib: require.resolve('browserify-zlib')
+        }
+      }
+    })
```

Comment:
We needed an instance of ProvidePlugin to polyfill the `Buffer` and `process` globals to make the CRA 5 upgrade work, and I suspect the same will be needed here. I'm not super familiar with storybook, but maybe we can just re-use the existing config decorator here -- something like `webpackFinal: require('../react-app-rewired.config.js').webpack` ?

Reply:
Not sure but the storybook build works fine with this config. Remember, it's not building our app, it's building a storybook app.